src/site/content/en/blog/economic-times-inp/index.md

```diff
@@ -0,0 +1,163 @@
+
+Because of all these efforts, we reduced TBT significantly, and our INP was reduced accordingly by almost 50%:
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/ojm0miY1lwoFqxZWnYVM.png", alt="A screenshot of the INP audit in CrUX. The INP for the page is 539 milliseconds, which exceeds the 'poor' threshold.", width="574", height="216" %}
+</figure>
+
+At this point, we nearly ran out of easy wins to further reduce TBT (and INP by proxy), but we knew we had a lot of room for improvement. This is when we decided to upgrade our custom-built UI boilerplate to the latest version of React along with [Next.js](https://nextjs.org/) to make better use of [hooks](https://react.dev/reference/react) to avoid unnecessary re-rendering of components.
+
+Due to more frequent updates and comparatively lesser traffic as compared to the other portions of the website, we began to migrate our [topic pages](https://economictimes.indiatimes.com/topic/home) to Next.js. We also used [PartyTown](https://partytown.builder.io/) for offloading additional heavy main thread work to web workers, along with techniques like `requestIdleCallBack` for deferring non-critical tasks.
+
+## How has improving INP helped The Economic Times?
+
+### Current TBT and INP on origin
+
+At the time of publishing this post, the TBT for our origin was 120 milliseconds, down from 3,260 milliseconds when we began our optimization efforts. Similarly, the INP for our origin was 257 milliseconds after our optimization efforts, down from over 1,000 milliseconds.
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/6cMvebBQKG2nhyc1eYeB.png", alt="A screenshot of the INP audit in CrUX. The INP for the page is 257 milliseconds, which is within the 'needs improvement' thresholds.", width="582", height="218" %}
+</figure>
+
+### INP CrUX trend
+
+The traffic received on topic pages represents a significantly smaller portion of overall traffic. Hence, it was an ideal place for experimentation. The CrUX results along with the business outcomes were very encouraging, and led us to expand our efforts across the entire website to reap further benefits.
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/pkPEYIQIeEDKdIKOCFSY.png", alt="A screenshot of INP distributions as visualized in CrUX over a period of four months, starting in July 2022 and ending in October 2022. Values within the 'poor' and 'needs improvement' thresholds declined somewhat, while the values within the 'good' threshold increased.", width="800", height="190" %}
+</figure>
+
+### Akamai mPulse TBT Analysis
+
+We use [Akamai mPulse](https://www.akamai.com/products/mpulse-real-user-monitoring) as our RUM solution. Though TBT is a lab metric, we observed a consistent decrease in TBT, clearly mapping to the outcomes of our efforts to reduce INP. As can be seen in the screenshot below, TBT values eventually dropped from approximately 5 seconds in to around 200 milliseconds in the field.
```

Comment:
Confused about this sentence. If mPulse is RUM, and TBT is lab, how are you measuring TBT in mPulse?

Not big enough to hold up publishing but one to follow up afterwards.

Reply:
Confirmed with some Akamai folks I'm with that it DOES measure TBT in the field.

Suggest slight tweak to the wording:

```suggestion
We use [Akamai mPulse](https://www.akamai.com/products/mpulse-real-user-monitoring) as our RUM solution, which measures TBT in the field. We observed a consistent decrease in TBT, clearly mapping to the outcomes of our efforts to reduce INP. As can be seen in the screenshot below, TBT values eventually dropped from approximately 5 seconds in to around 200 milliseconds in the field.
```